massgov/openmass
# Update to core-recommended and a few media contrib modules

We remove `drupal/console` as its unused and not D9 compatible.

When using this branch locally, delete the /vendor directory before running `composer install`. This helps composer update to the right version for all dependencies.

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-19198


**To Test:**
- This PR is pushed to Dev environment.
- [ ] Test the updated contrib modules. 
  - For Media Entity Actions, see the last 4 actions in the Actions dropdown at https://edit.mass.gov/admin/content/media
  - Try uploading an image and cropping it. 
  - Try embedding an existing image using WYSIWYG
  - Try embedding a video and viewing it after saving.
  -  Upload a file and keep it private. Make sure a different editor view it and download it.
  - Click link icon in WYSIWYG and make a link to existing content
- [ ] To use drupal rector, run `vendor/bin/rector process --docroot/modules/custom/mass_utility` (for example)

Reroll: `composer update --with-all-dependencies drupal-composer/drupal-scaffold drupal/core-recommended drupal/core-composer-scaffold  "symfony/*" cweagans/composer-patches composer/installers drupal/core christian-riesen/otp aragonie/random_compat drupal/console  drupal/upgrade_status drupal/upgrade_rector drupal/auto_entitylabel drupal/imagick drupal/crop drupal/video_embed_field drupal/private_files_download_permission drupal/linkit drupal/file_browser`

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
